src/MICore/project.json

```diff
@@ -12,7 +12,8 @@
     "System.Reflection.TypeExtensions": "4.0.0",
     "System.Security.Cryptography.X509Certificates": "4.0.0-beta-23225",
     "System.Threading.Thread": "4.0.0-beta-23225",
-    "System.Xml.XmlSerializer": "4.0.11-beta-23225"
+    "System.Xml.XmlSerializer": "4.0.11-beta-23225",
+    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-beta-23225"
```

Comment:
You also need to add this to C:\MIEngine\src\RequiredNetFX46FacadeAssemblies.targets.